package-lock.json

```diff
@@ -272,7 +272,7 @@
 				"qs": "^6.9.1",
 				"redux": "^4.0.4",
 				"tslib": "^1.10.0",
-				"wpcom-proxy-request": "file:packages/wpcom-proxy-request"
+				"wpcom-proxy-request": "file:https:/registry.npmjs.org/wpcom-proxy-request/-/wpcom-proxy-request-5.0.2.tgz"
```

Comment:
what in the world. @jsnajdr @ockham just fyi npm is weird.

Reply:
reverted this commit, but I got here by taking the first commit and running `npm dedupe && npm i`